server/chat-formatter.ts

```diff
@@ -444,3 +444,14 @@ class TextFormatter {
 export function formatText(str: string, isTrusted = false) {
 	return new TextFormatter(str, isTrusted).get();
 }
+
+/**
+ * Takes a string and strips all standard chat formatting from it, the text of a link is kept.
```

Comment:
Should probably add the comment about greentext here, otherwise it's kind of misleading.